test/Driver/linker.swift

```diff
@@ -108,6 +111,25 @@
 // LINUX-x86_64-DAG: -Xlinker -undefined
 // LINUX-x86_64: -o linker
 
+// LINUX-armv6: swift
+// LINUX-armv6: -o [[OBJECTFILE:.*]]
+
+// LINUX-armv6: clang++{{"? }}
+// LINUX-armv6-DAG: [[OBJECTFILE]]
+// LINUX-armv6-DAG: -lswiftCore
+// LINUX-armv6-DAG: -L [[STDLIB_PATH:[^ ]+/lib/swift]]
+// LINUX-armv6-DAG: --target=junk-unknown-linux-gnueabihf
```

Comment:
When lit is running a test on a file with more than one test (such as linker.swift), it looks like it bails when it finds the first failure, right?  I deleted the tests relating to macos and the relative linker and verified that the tests for providing --target to clang functions as intended.